packages/swingset-liveslots/src/virtualObjectManager.js

```diff
@@ -680,21 +675,53 @@ export function makeVirtualObjectManager(
     }
     // beyond this point, we use 'multifaceted' to switch modes
 
-    if (isDurable) {
-      checkAndUpdateFacetiousness(
-        tag,
-        durableKindDescriptor,
-        facetNames,
-        saveDurableKindDescriptor,
-      );
-    }
+    // The 'stateShape' pattern constrains the `state` of each
+    // instance: which properties it may have, and what their values
+    // are allowed to be. For durable Kinds, the stateShape is
+    // serialized and recorded in the durableKindDescriptor, so future
+    // incarnations (which redefine the kind when they call
+    // defineDurableKind again)
```

Comment:
Missing end of comment?

Reply:
oops, good catch, I'll finish that thought